README.md

```diff
@@ -21,31 +21,158 @@ npm install
 ```
 npm start
 ```
-The Admin App is now served on [localhost](http://localhost:3000). When you first open the URL, you are prompted to configure the Sitefinity CMS instance you are working with. In the URL field, enter the instance details and then save the configuration. You can later change the configuration by navigating to [config] (http://localhost:3000/config). 
-
-Once you setup the Sitefinity CMS instance, the server becomes in watch mode and automatically recompiles and serves any newly created files.
 
 3. When you are done developing execute the following command:
 ```
 npm run build
 ```
 As a result, a JavaScript file (**extensions.bundle.js**) is generated in the **dist** folder.
 
-4. Register your extensions with the Admin App by upload this file to the **extensions.bundle.js** file in the **root** of the Sitefintiy CMS web application and then refresh your Sitefinity CMS page.
+4. Register your extensions with the Admin App by uploading this file to the **extensions.bundle.js** file in the **root** of the Sitefintiy CMS web application and then refresh your Sitefinity CMS page.
+
+
+### Configure Sitefinity CMS for development of custom extensions
+
+To enable Sitefinity CMS to recognize and allow working with the Admin App, you need to configure the following:
+
+
+#### STS configuration
+1. Navigate to *Administration* -> *Settings* -> *Advanced* -> *Authentication* -> *SecurityTokenService* -> *IdentityServer* -> *Clients* -> *sitefinity*.
+2. Under *Allowed cors origins*, click *Create new*.
```

Comment:
@Gebov I was able to get the extensions working only with the two CORS related configurations. STS points 1 to 3 and Web service configuration.

Reply:
lets leave them as they are for now